book/website/collaboration/remote-collab/remote-collab-code-review.md

```diff
@@ -0,0 +1,34 @@
+* Trying to learn a new language and not knowing the most efficient way to solve a given problem. Others might have better ways to solve that problem which you might not know about. 
+* Getting feedback about our code's interface, in order to make it more usable.
+* How do you get this feedback while being considerate of other people's time?
+* Getting started and gauging interest in a group code review
+
+## 2. Solutions & Recommendations
+* Code with comprehension and (try to)  make the code reviewing process easier by being clear and organised with the format and logic. The code itself does not have to look professional due to limited experience of the coder, but should be equipped with comments that help reviewers to understand. Good format to allow advanced coding (Object-oriented programming, list comprehension) to be applied.  
+* Be as specific as you can in terms of the aspects of your code that need review. Is it a specific function that you are worried about? Or perhaps the way you have structured your code?
+* Ask someone else to use your library/run your code, even for a toy example. If they find it hard or ask you questions on how to use it, then there is room for improvement :smile: 
+* Develop code with other operating systems in mind where possible (for example, specify your OS and environment settings) and include code run on binder to ensure others can run the code in some form - testing code on multiple operating systems also provides a better opportunity for reproducibility testing!
+* Contacting your team for gauging interest (for example, via email, raising the topic in lab meetings or sharing a poll) alongside your institution for any more experienced recommendations on how to get started with a group code review - for example, university admin teams may have setup similar groups before. Also include people interested and institution experts in your prelimnary meetings to setup the code review group.
+
+
+## 3. Examples
+* Scenario: live code-review session with the research group.
+* The person asking for code review should share the code in advance, together with some specific questions that they'd like to be addressed [(Reference)](https://the-turing-way.netlify.app/reproducible-research/reviewing/reviewing-recommend.html).
+* Mark the specific chunk of codes that require reviewing, so to save time for the reviewers. Differenciate the parts for change. 
+* Prioritise the chunks which require urgent updates and feedback. 
+
+## 4. Summary
+* Code review is awesome! However it can seem daunting at first to put yourself out there. We are not often encouraged to share our work and ask for feedback. But doing so can have two main benefits:
+    1. Help you improve your code by getting tips and tricks for multiple people.
+    2. Your code will inevitably become more reproducible as a result of you having to think about people, other than yourself, reading your code.
+
+**References**
```

Comment:
This heading should be "Further Reading" and the chapters in the Turing Way should be updated according to the style guide: https://the-turing-way.netlify.app/community-handbook/style/style-crossref.html

You may also want to add some more context on why the links below are useful - do they provide more resources on organising code reviews or for people preparing for code reviews?